### PR TITLE
[3] prepare for v2.0.0

### PR DESCRIPTION
**Pull request by trel**

### CODE REVIEW

**Review by korydraughn — APPROVED** [review]
Looks good.

Pound when ready.

File: setup.py [hunk 1]
```diff
@@ -27,11 +27,16 @@
...
+                   'Development Status :: 5 - Production/Stable',
                    'Programming Language :: Python :: 2.7',
                    'Programming Language :: Python :: 3.4',
                    'Programming Language :: Python :: 3.5',
                    'Programming Language :: Python :: 3.6',
+                   'Programming Language :: Python :: 3.7',
+                   'Programming Language :: Python :: 3.8',
+                   'Programming Language :: Python :: 3.9',
+                   'Programming Language :: Python :: 3.10',
+                   'Programming Language :: Python :: 3.11',
```
Comment (alanking) [inline setup.py:39]: Have we tried this on 3.12?

Comment (trel) [reply setup.py:39]: i wrote the new password booth on 3.12.  so, yes!

Comment (trel) [reply setup.py:39]: added

**Review by korydraughn — APPROVED** [review]
Squash/Pound it.

**Comment by trel** [comment]
going to wait until a couple more dan PRs are ready/merged.

**Review by alanking — APPROVED** [review]

**Pull request merged**